src/EFCore.PG/Extensions/NpgsqlEntityTypeBuilderExtensions.cs

```diff
@@ -109,6 +109,43 @@ public static class NpgsqlEntityTypeBuilderExtensions
 
         #endregion Comment
 
+        #region Unlogged Table
+
+        /// <summary>
```

Comment:
We can add a link to https://www.postgresql.org/docs/current/sql-createtable.html#SQL-CREATETABLE-UNLOGGED in the <remarks> section.